static/js/webrtc.js

```diff
@@ -314,6 +322,15 @@ var rtc = (function() {
         .append($largeVideo)
         .insertAfter($video);
     },
+    // Sends a stat to the back end. `statName` must be in the
+    // approved list on the server side.
+    sendErrorStat: function(statName) {
+      var msg = { "component" : "pad", // FIXME: Remove this stupidity!
```

Comment:
Comment copied from etherpad's code. Is there a reason I would need this until some plumbing gets fixed? Or can I remove this now?

Reply:
I guess I'll just remove the FIXME comment. Not sure what it's about.